doc/source/point-in-time-recovery.rst

```diff
@@ -77,15 +77,54 @@ Re-enable |PITR| to resume saving oplog slices:
    $ pbm config --set pitr.enabled=true
 
 
-.. rubric:: Delete a backup
+Delete backups
+========================
 
-When you :ref:`delete a backup <pbm.backup.delete>`, all oplog slices that relate to this backup will be deleted too. For example, you delete a backup snapshot 2020-07-24T18:13:09 while there is another snapshot
-2020-08-05T04:27:55 created after it.  |pbm-agent| deletes only oplog slices that relate to 2020-07-24T18:13:09.
+As of version 1.6.0, backup snapshots and incremental backups (oplog slices) are deleted using separate commands: :command:`pbm delete-backup` and :command:`pbm delete-pitr`  respectively.
 
-The same applies if you delete backups older than the specified time.
+Running :command:`pbm delete-backup` deletes any backup snapshot but for the following ones:
+
+-  A backup that can serve as the base for any point in time recovery and has |PITR| time ranges deriving from it
+-  The most recent backup if |PITR| is enabled and there are no oplog slices following this backup yet.
+
+To illustrate this, let’s take the following ``pbm list`` output:
+
+.. code-block:: bash
+
+   $ pbm list
+   Backup snapshots:
+   2021-07-20T03:10:59Z [complete: 2021-07-20T03:21:19]
+   2021-07-21T22:27:09Z [complete: 2021-07-21T22:36:58]
+   2021-07-24T23:00:01Z [complete: 2021-07-24T23:09:02]
+   2021-07-26T17:42:04Z [complete: 2021-07-26T17:52:21]
+
+   PITR <on>:
+   2021-07-21T22:36:59-2021-07-22T12:20:23
+   2021-07-22T12:32:36-2021-07-25T19:16:09
```

Comment:
```suggestion
```
this timeline is impossible since its start time ain't derive from any available backup